sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/upload_from_file.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import tempfile
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from ._test_base import _FileTest
+
+
+class UploadFromFileTest(_FileTest):
+    temp_file = None
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = get_random_bytes(self.args.size)
+        with tempfile.NamedTemporaryFile(delete=False) as temp_file:
+            UploadFromFileTest.temp_file = temp_file.name
+            temp_file.write(data)
+
+    async def global_cleanup(self):
+        os.remove(UploadFromFileTest.temp_file)
+        await super().global_cleanup()
+
+    def run_sync(self):
+        with open(UploadFromFileTest.temp_file, 'rb') as fp:
```

Comment:
Same feedback as blob, is uploading from file worth testing?

Reply:
I'm happy to remove it if we don't feel need to test parity against the T1 API for this.
We can also remove if from the table of results in onenote.

Reply:
Let's keep it for now